CMakeLists.txt

```diff
@@ -27,7 +27,6 @@ option(BUILD_ROS_INTERFACE "enable ROS subscriber for motor failure plugin" "OFF
 ## System dependencies are found with CMake's conventions
 find_package(PkgConfig REQUIRED)
 find_package(gazebo REQUIRED)
-find_program(px4 REQUIRED)
```

Comment:
Do you remove this b/c you launch px4 separately?

Reply:
ah I see your commit message on #4 explains: " Removed px4 cmake package. Px4 should be built and launched separately "